runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldTableHeader.java

```diff
@@ -45,18 +45,26 @@
 
 class WorldTableHeader extends JPanel
 {
-	private static final ImageIcon ARROW_DOWN;
 	private static final ImageIcon ARROW_UP;
-	private static final ImageIcon ARROW_UP_FADED;
+
+	private static final Color ARROW_COLOR = ColorScheme.LIGHT_GRAY_COLOR;
+
+	private static final ImageIcon HIGHLIGHT_ARROW_DOWN;
+	private static final ImageIcon HIGHLIGHT_ARROW_UP;
+
+	private static final Color HIGHLIGHT_COLOR = new Color(210, 193, 53);

```

Comment:
What was the rationale behind this color choice? Is there a preference for this color to `ColorScheme.BRAND_ORANGE`?

Also, the ImageIcons and Colors should be grouped next to each other.

Reply:
The color was the suggested color in #7022. Would you prefer I changed it to `ColorScheme.BRAND_ORANGE`?

Reply:
I suppose it would be worth checking, seeing as that would help to be more consistent across the UI.

Reply:
I like it better. Done and gif updated